documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -122,6 +122,24 @@ kubectl label secret _CA-KEY-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_C
 * Label `strimzi.io/kind=Kafka` identifies the Kafka custom resource.
 * Label `strimzi.io/cluster=_CLUSTER-NAME_` identifies the Kafka cluster.
 
+. Annotate the secrets
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-CERTIFICATE-SECRET_ strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_
+----
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-KEY-SECRET_ strimzi.io/ca-key-generation=_CA-KEY-GENERATION_
+----
++
+* Annotation `strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_` defines the generation of a new CA certificate.
+* Annotation `strimzi.io/ca-key-generation=_CA-KEY-GENERATION_` defines the generation of a new CA key.
++
+If you are replacing CA certificates automatically generated by the Cluster Operator, use the next higher incremental value from the existing annotation.
```

Comment:
I'm confused. Does this refer to migration from Strimzi generated CA to custom CA?

Reply:
Yes. I thought it was important to highlight this if user starts with Strimzi CAs and then wants to move to his own CA certs. For triggering the update, increasing the generation is needed.